src/qml/editorwidgets/EditorWidgetBase.qml

```diff
@@ -1,11 +1,28 @@
 import QtQuick 2.14
+import QtQuick.Controls 2.14
 
 Item {
     /* This property indicates whether the editor widget has been fully loaded by its Loader.
      * Note: prior to this property being true, signals emitted by the editor widget will not be
      * propagated.
      */
     property bool isLoaded: false
+    property bool hasMenu: false
+    property Menu menu: Menu {
+      id: itemMenu
+      title: qsTr( "Item Menu" )
+
+      width: {
```

Comment:
Interesting ...
Isn't that done by the framework itself?

Reply:
@m-kuhn , not AFAIK, that's a trick I implemented for all of our Menu {} item, as per "the Internet"'s recommendation :)

Reply:
Would a `QfMenu` implementation help?

Reply:
@m-kuhn , hmm, I'm not sure how that'd work out when we need to add MenuItem et cie within the custom QfMenu. In any case, let's explore and experiment that possibility in separate PR? It'd involved changing a bunch of QML files anyways.

Reply:
yes, an isolated PR for this (if reasonably feasible) is the correct approach.